repositories.json

```diff
@@ -2,4 +2,4 @@
   "name" : "QuAntiL Types",
   "url" : "https://github.com/UST-QuAntiL/tosca-definitions-quantil",
   "branch" : "main"
-}
```

Comment:
You should not rely on the dependencies that come with this repository. Explicitly state the dependency to the commons and normative repositories here.

Reply:
Alright (if you are sure that this does not result in multiple clones of the dependencies :D )

Reply:
Yes, we found the problem 😉